connect/api/src/main/java/org/apache/kafka/connect/connector/ConnectRecord.java

```diff
@@ -140,7 +140,9 @@ public String toString() {
                 "topic='" + topic + '\'' +
                 ", kafkaPartition=" + kafkaPartition +
                 ", key=" + key +
+                ", keySchema=" + (keySchema != null ? "..." : "null") +
                 ", value=" + value +
+                ", valueSchema=" + (valueSchema != null ? "..." : "null") +
```

Comment:
I would consider something other than just `...`. It took me some time to figure out why we were even adding this. `non-null` or something like that, or changing the format so it isn't `keySchema=` which implies (to me at least) that we're putting the actual value. Alternatively, you can include the actual schema, currently it's `toString()` just returns name & type, not a full, nested representation of the schema.

Reply:
I agree with your suggestion to use the `Schema.toString()`